Userland/Libraries/LibCore/MimeData.cpp

```diff
@@ -54,164 +54,115 @@ void MimeData::set_text(DeprecatedString const& text)
     set_data("text/plain", text.to_byte_buffer());
 }
 
-static AK::Array<StringView, 8> s_plaintext_suffixes = {
+// FIXME: Share this, TextEditor and HackStudio language detection somehow.
```

Comment:
Commit message typos:
- _used to have to separate_ -> _used to have two separate_
- _fill in some gap_ -> _fill in some gaps_